### PR TITLE
Doodlintoolbar

### PR DESCRIPTION
**Pull request by nateirwin25**

### CODE REVIEW

File: src/annotator/components/toolbar.js [hunk 1]
```diff
@@ -100,51 +103,101 @@ export default function Toolbar({
...
+            expanded={isSidebarOpen}
+            onClick={toggleSidebar}
+          />
+        )}
+        {!useMinimalControls && (
+          <div className="annotator-toolbar-buttonbar">
+            <ToolbarButton
+              label="Stop doodle"
+              icon="close"
+              onClick={drawingToolbarToggle}
```
Comment (WiFuchs) [inline src/annotator/components/toolbar.js]: This function should set the doodleability to False

File: src/annotator/components/toolbar.js [hunk 2]
```diff
@@ -100,51 +103,101 @@ export default function Toolbar({
...
+            />
+            <ToolbarButton
+              label="Pen"
+              icon="pen"
+              onClick={toggleDoodleability}
```
Comment (WiFuchs) [inline src/annotator/components/toolbar.js]: this should just set the tool to pen now, not toggle the doodleability, but I can add that when I add the eraser tool.

File: src/annotator/components/toolbar.js (hunk 1 above)
Comment (WiFuchs) [reply src/annotator/components/toolbar.js]: Or just use the `toggleDoodleability` function

**Review by smjaques — APPROVED** [review]
Looks good

**Review by WiFuchs — COMMENTED** [review]
Looks good except closing the toolbar doesn't change the doodleability of the canvas.

**Review by WiFuchs — APPROVED** [review]
lgtm!

**Pull request merged**